templates/kubernetes/index.html

```diff
@@ -6,409 +6,379 @@
-    </div>
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+    <h2 class="p-text--small-caps">Fully-supported Kubernetes clouds</h2>
   </div>
-  <div class="row">
-    <div class="p-card col-6">
-      <p class="p-card__title">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/03617f57-MicroK8s-logo-2022_RGB.svg",
-          alt="",
-          width="150",
-          height="45",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
-      </p>
-      <p class="p-heading--5">Low-touch Kubernetes for clouds, edge and IoT</p>
-      <div class="p-card__content">
-        <div class="u-sv3 u-hide--small u-hide--medium u-align--center">
-        {{
-          image(
-          url="https://assets.ubuntu.com/v1/9c55d397-IoT-Gateways-Kubernetes2.svg",
-          alt="",
-          width="400",
-          hi_def=True,
-          loading="lazy",
-          ) | safe
-        }}
-        </div>
-        <ul class="p-list">
-          <li class="p-list__item is-ticked">All Kubernetes services and the most popular add ons in a single package</li>
-          <li class="p-list__item is-ticked">Minimal ops for easy clustering, self-healing HA and simpler upgrades</li>
-          <li class="p-list__item is-ticked">Lightweight K8s for resource-constrained environments</li>
-          <li class="p-list__item is-ticked">Secure by default, strict confinement, and long term support</li>
-          <li class="p-list__item is-ticked">Fixed-price deployment</li>
-          <li class="p-list__item is-ticked">Compatible with Linux, Windows and macOS</li>
-        </ul>
-        <ul class="p-inline-list">
-          <li class="p-inline-list__item"><a href="https://microk8s.io/">Try MicroK8s today&nbsp;&rsaquo;</a></li>
-          <li class="p-inline-list__item"><a href="https://microk8s.io/resources">Learn more about MicroK8s&nbsp;&rsaquo;</a></li>
-        </ul>
-      </div>
-    </div>
-    <div class="p-card col-6">
-      <p class="p-card__title">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/e6cd15f7-Charmed+Kubernetes_RGB_2022.svg",
-          alt="",
-          width="300",
-          height="45",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
-      </p>
-      <p class="p-heading--5">Composable, operator-based Kubernetes for the enterprise</p>
-      <div class="p-card__content">
-        <div class="u-sv3 u-hide--small u-hide--medium u-align--center">
-          {{
-            image(
-            url="https://assets.ubuntu.com/v1/5a71594a-updated-charmed-k8s-calico.svg.svg",
-            alt="",
-            width="320",
+  <div class="u-fixed-width">
+    <div class="p-logo-section">
+      <div class="p-logo-section__items">
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/3948d6dd-openstack-logo.png",
+            alt="OpenStack",
```

Comment:
Some of the logos differ from the ones in the copy doc. @Sophie-32 are the current logos correct?

Reply:
Design is the source of truth for images, doc is probably out of date